lib/kafo/kafo_configure.rb

```diff
@@ -525,8 +525,8 @@ def run_installation
               @progress_bar.update(line) if @progress_bar
             end
           rescue Errno::EIO # we reach end of input
-            exit_status = PTY.check(pid, true) if PTY.respond_to?(:check) # ruby >= 1.9.2
-            if exit_status.nil? # process is still running or we have old ruby so we don't know
+            exit_status = PTY.check(pid, true) if PTY.respond_to?(:check)
```

Comment:
Feels like you can just make it like this on Ruby 1.9.2+
```suggestion
            exit_status = PTY.check(pid, true)
```